products/BellHybrid/apps/common/include/common/data/FrontlightUtils.hpp

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) 2017-2022, Mudita Sp. z.o.o. All rights reserved.
+// For licensing, see https://github.com/mudita/MuditaOS/LICENSE.md
+
+#pragma once
+
+#include <cmath>
+#include <cstdint>
+
+namespace frontlight_utils
+{
+    namespace
+    {
+        static constexpr auto minPercent                       = 0.0f;
+        static constexpr auto maxPercent                       = 100.0f;
+        static constexpr auto minimumLightOnPercentOffsetValue = 16.0f;
+        static constexpr auto minBrightness                    = 1U;
+        static constexpr auto maxBrightness                    = 10U;
+        static constexpr float multiplier = (maxPercent - minimumLightOnPercentOffsetValue) / maxBrightness;
+    } // namespace
+
+    /// 1-10 range
+    using Brightness = std::uint8_t;
+
+    static inline float fixedValToPercentage(Brightness value)
```

Comment:
Maybe those two functions could be simplified with `std::clamp()` or something alike?

Reply:
In this case `std::clamp()` is not useful at all, but thanks for your advice :) I think that `std::min` and `std::max` is fine here so I'm leaving it as it is.